frb_dart/lib/src/basic.dart

```diff
@@ -209,3 +209,21 @@ class WireSyncReturnStruct extends ffi.Struct {
   @ffi.Uint8()
   external int success;
 }
+
+/// An opaque pointer to a native C or Rust type.
+/// Recipients of this type should call [dispose] at some point during runtime.
+class Opaque {
```

Comment:
This name might clash with https://api.flutter.dev/flutter/dart-ffi/Opaque-class.html